src/data/usage-data-service.js

```diff
@@ -14,7 +14,7 @@ const UsageDataService = {
    * @param {Parameters<typeof ApiService['post']>[1]} data
    */
   post(endpoint, data) {
-    return ApiService.post(`analytics/${endpoint}/`, data)
+    return ApiService.post(`analytics/${endpoint}`, data)
```

Comment:
:sweat_smile: At some point I think I added this trailing slash to silence a console warning about it missing. Does that console warning not happen any more?

Reply:
The warning now was because it was duplicated.